### ISSUE TITLE
Simple distribution tracking

### ISSUE THREAD

**Issue by Koeng101** [vsoch/freegenes#19]
As an admin, it would be very useful for us to be able to see where distributions have been shipped. 

For example, if you browse distributions as an admin, you could view how many of each distribution have been shipped to people, and when you click on the the shipment number, led to a list of places that distribution has been shipped

**Comment by vsoch** [comment]
You know what would be really cool too? If we had a map (even country or state based) that showed the counts. Good call on the "enhancement" label, although it sounds really fun so I'm definitely wanting to do it (todo label).

**Comment by vsoch** [comment]
@Koeng101 do you have ideas for how we could do this without saving the addresses in the database? if they are kept with shippo, then we could iterate through the orders that have labels, and then look up the transaction / shipping info using the shippo API for all orders that quality. We can then generate a map from that. Would this be expensive to do, for each time we want to generate the map?

**Comment by Koeng101** [comment]
@vsoch perhaps it can just be updated hourly or daily - though I don't know the difficulty of implementing that.

**Comment by vsoch** [comment]
That would be very easy to do - we would use the API to get updated counts, and then reveal some granularity that doesn't directly identify anyone. I'll work on this after the development server is up and running.

**Comment by vsoch** [comment]
@Koeng101 what is the granularity of your orders? For example, would a map with states highlighted suffice? or are you supporting international shipping? It would be nicest looking (imho) to have a state map with states colored based on counts, but if you have international, we could do a standard world map with points set at cities where orders come from.

**Comment by vsoch** [comment]
hey @Koeng101 one more note for you before I'm heading out for the day - I'm creating the function to download the data, and it looks like for the orders (to get the complete information) I need an actual API key - would you send one to me securely, with absolute certainty that I won't use it for creating / amending orders, just getting the listing? Or another option is for you to issue the request and send me the data (but then we would be sending personal data, which maybe isn't the best idea). If you do this - please sent to my email with SECURE in the title. What we basically need to do is:

```python
import requests                                                                                                                                       
url="https://api.goshippo.com/orders/"  
headers={"Authorization": "ShippoToken %s" % SHIPPO_TOKEN} 
response = requests.get(url, headers=headers)

# This should have some entities I think, but doesn't because it's the testing API
response.json()
{'next': None, 'previous': None, 'results': []}
```

So - let me know which works better! I can either get a secure API key, or you can make the request. If you trust me with an API key, I will likely use it for testing and then dispose until we need on the server.

**Comment by vsoch** [comment]
okay, no need for the token (at least before we want to test!) What I did is use the sample data to create some dummy data (and based the export function on that) and then generated the map view for orders:

![image](https://user-images.githubusercontent.com/814322/64262916-d4df5f80-cefc-11e9-828c-dc6c88ab8a54.png)

It's based on latitude and longitude from zip codes, and the size of the marker is based on the number of orders. Likely we'll need to tweak the scale when we have real data, but this should be good to start. I added the dummy data to the server so there is always _some_ map there, even without running the task.
